command/apply_test.go

```diff
@@ -182,13 +185,15 @@ func TestApply_parallelism(t *testing.T) {
 
 	par := 4
 
-	// This blocks all the apply functions. We close it when we exit so
-	// they end quickly after this test finishes.
-	block := make(chan struct{})
-	// signal how many goroutines have started
-	started := make(chan int, 100)
+	// started is a semaphore that we use to ensure that we never have more
+	// than "par" apply operations happening concurrently
+	started := make(chan struct{}, par)
 
-	runCount := &hwm{}
```

Comment:
I think the `hwm` struct is now unused and maybe could be removed?

Reply:
Thanks, I meant to do that!